metrics/scaling/k8s_scale.sh

```diff
@@ -67,6 +67,15 @@ EOF
 	# use 3 for the file descriptor rather than stdin otherwise the sh commands
 	# in the middle will read the rest of stdin
 	while read -u 3 name node; do
+		# look for taint that prevents scheduling
```

Comment:
first part of filtering out NoSchedule nodes from later pod / GB and pod / CPU calculation. Currently master is calculated in even if NoSchedule taint is set.